09-intermediate-rails/01-session-and-oauth.md

```diff
@@ -151,6 +151,7 @@ class User < ActiveRecord::Base
     user.name  = auth_hash['info']['name']
     user.email = auth_hash['info']['email']
 
+    user.save
```

Comment:
This isn't in the model method b/c it's in the Controller action instead - and will show a different page based on whether or not it saved correctly